site/docs/configuration/expected-outputs/guardrail.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 101
+sidebar_label: Guardrails
+---
+
+# Guardrails
+
+Use the `guardrails` assert type to ensure that LLM outputs pass safety checks based on the provider's built-in guardrails.
+
+This assertion checks if the provider's response includes guardrails information and verifies that the content hasn't been flagged for safety concerns.
```

Comment:
this sentence doesn't say much. You could mention how guardrails can exist on both the input and output. input often check for things like prompt injections, output often check for harm categories. 

mention which providers support this around here.